<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="Canvas" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>0</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="startXField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="startYField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="moveLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Черчение отрезка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>0</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Черчение спектра</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>861</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начало отрезка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="endXField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endYField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Конец отрезка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>239</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_6">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алгоритм</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="lineAlgorithmComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>269</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Брезенхем (Целочисленный)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхем (Вещественный)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхем (С устранением ступенчатости)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цифровой дифференциальный анализатор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ву</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Функция стандартной библиотеки</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="lineDrawButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начертить</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>299</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_7">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="lineColorComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Желтый</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="rangeColorComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Желтый</string>
     </property>
    </item>
   </widget>
   <widget class="QSplitter" name="splitter_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_8">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алгоритм</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="rangeDrawButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начертить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rangeAlgorithmComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>579</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Брезенхем (Целочисленный)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхем (Вещественный)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхем (С устранением ступенчатости)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цифровой дифференциальный анализатор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ву</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Функция стандартной библиотеки</string>
     </property>
    </item>
   </widget>
   <widget class="QSplitter" name="splitter_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>609</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_9">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Длина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Угол рассеивания</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rangeLengthField">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rangeAngleField">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_author">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Об авторе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prog">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
